client/src/models/services/queries/new_service_queries.ts

```diff
@@ -0,0 +1,29 @@
+import { lang } from "src/core/injected_build_constants";
+
+import { useSuspensedQuery } from "src/graphql_utils/useSuspensedQuery";
+
+import type {
+  ServicesForOrgQuery,
+  ServicesForOrgQueryVariables,
+} from "./ServicesForOrg.gql";
+import { ServicesForOrgDocument } from "./ServicesForOrg.gql";
+import type {
+  ServicesForProgramQueryVariables,
+  ServicesForProgramQuery,
+} from "./ServicesForProgram.gql";
+import { ServicesForProgramDocument } from "./ServicesForProgram.gql";
+
+export const useServicesForOrg = (id: string) => {
+  const { data } = useSuspensedQuery<
+    ServicesForOrgQuery,
+    ServicesForOrgQueryVariables
+  >(ServicesForOrgDocument, { lang, id });
+  return data?.root.org?.services;
+};
+export const useServicesForProgram = (id: string) => {
+  const { data } = useSuspensedQuery<
+    ServicesForProgramQuery,
+    ServicesForProgramQueryVariables
+  >(ServicesForProgramDocument, { lang, id });
+  return data?.root.program?.services;
+};
```

Comment:
This is the new recipe for data-fetching. Each query involves importing the query document, as well as the query type and its variables' type. 

There's an extra layer here, because both of these queries were used twice, but you'll often use `suspenseQuery` directly in components if you'd like. I extracted these into their own hooks. 

What's really neat is that you can split up or combine these hooks, making them their own "layer". I haven't combined queries yet, but you'll likely want to create a new hook [that uses `useQueries`](https://react-query.tanstack.com/guides/parallel-queries#dynamic-parallel-queries-with-usequeries), because in suspense mode two `useQuery` hooks side-by-side will not be executed in parallel.